charts/velero/templates/cleanup-crds.yaml

```diff
@@ -53,6 +53,10 @@ spec:
               kubectl delete volumesnapshotlocation --all;
               kubectl delete podvolumerestore --all;
               kubectl delete crd -l app.kubernetes.io/name=velero;
+          {{- with .Values.kubectl.resources }}
```

Comment:
Should these additions be put behind a conditional so that empty "resources" fields aren't added? I'm not 100% sure that it matters, but that way there's no change in yaml for cases where the optional resources aren't specified:
```
    {{- if .Values.kubectl.resources }}
```

Reply:
Hi,
 thanks for the review.

I had the same concern but testing with helm template if resources is empty (either `resources: {}` like in values.yaml or simply `resources:`) nothing gets included in the final manifest. Basically, if there's something defined it gets converted in yaml and indented with 12 spaces; if there's nothing, nothing gets added as expected. 

The same style of inclusion is used freely in the chart ( e.g. for the resources of the main deployment here https://github.com/vmware-tanzu/helm-charts/blob/0c49944a8ee8ae9ac92a508e0344a2249d05fafa/charts/velero/templates/deployment.yaml#L112, the same goes for nodeSelector,affinity, tolerations, etc.).

Reply:
I prefer using `with` syntax personally.